### DISCUSSION TITLE
How to make arbitrage bot?

### DISCUSSION THREAD

**Discussion by 0xlightfury** [community/community#40667] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I have build own dex(https://lightfury-swap.netlify.app/) and price chart app((https://lightfury-chart.netlify.app/) on goerli testnet. There are 3 pairs (BUSD/ETH, FURY/ETH, FURY/BUSD). If I buy FURY token from ETH, the FURY price is increased in FURY/ETH pair. But the FURY price in FURY/BUSD is not changed according to it. I think this action will be done by arbitrage bot. I hope anyone help me in this field or cooperate with me.
Thanks.

**Comment by jfullstackdev** [comment]
there are so many bots on trading and crypto, you may want to check them out

https://github.com/topics/trading-bot

once you have your pick, you can start collaborating with them

**Reply by 0xlightfury** [reply]
Ok, Thank you for your attention. I will check them all.
